CONTRIBUTING.md

```diff
@@ -63,30 +63,66 @@ adding `--release` flag.
   Keep in mind, that only one JVM can be run per process. Therefore, tests that
   need to launch it with different parameters have to be placed in different 
   source files. `Cargo` runs tests from different modules in parallel.
+
+* Benchmarks are in [benches](benches) directory. As integration tests, they
+  require launching a JVM from native.
+
 * Doc tests are rarely used, but they allow to efficiently test some functionality
   by providing an example of its usage. Consult 
   [Rust documentation](https://doc.rust-lang.org/beta/rustdoc/documentation-tests.html)
   for more info about writing these tests.
 
 ### Running Tests
 
-To run all tests, you should execute the following command:
+#### Setup Environment
+
+As some tests need to launch a JVM, add a directory with JVM library that you want
+to use to `LD_LIBRARY_PATH` on Linux/Mac or `PATH` environment variable on Windows.
+On Linux/Mac, you can use the following script for this purpose:
+
+```$sh
+JAVA_HOME="${JAVA_HOME:-$(java -XshowSettings:properties -version \
+    2>&1 > /dev/null |\
+    grep 'java.home' |\
+    awk '{print $3}')}"
+LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
+
+export LD_LIBRARY_PATH="${LIBJVM_PATH}"
+```
+
+#### Run Tests
+
+To run all tests, execute the following command:
 
 ```$sh
-$ cargo test --features=backtrace,invocation
+cargo test --features=backtrace,invocation
 ```
 
 This command will run all tests, including unit, integration and documentation
 tests.
 
+#### Run Benchmarks
+
+To run all benchmarks, execute the following command (nightly Rust required):
```

Comment:
We can actually rewrite benches using `criterion` library which doesn't need nightly

Reply:
Sounds interesting — is it a drop-in replacement for the standard tool? Is it also more feature-rich (the present framework seems utterly inferior to, say, JMH from Java)?

Reply:
The changes a pretty straightforward. https://github.com/kvark/froggy/pull/71/commits/80891edf343d842804281d1327e628f2b9f1adc7
It also seems to be more feature-reach, comparing to the standard benches, especially when talking about gathering statistics and making plots and reports